### ISSUE TITLE
Create Experimental Build Worker and Recreate Per Commit Main Sketch Build Worker

### ISSUE THREAD

**Issue by CodexLink** [CodexLink/GeM-ESSAC#26]
This was intended to ensure that we have everything working. Keep in mind that, our base works must be the Sketch Build Worker itself. So that means, we have to filter a lot of files that correlate to their code tests.

I gotta do some modifications first since we're going into the returnee state of this project. Here's the list:

- [x] Recreate File Named Experimental Build Worker (the filename may vary).
- [x] Learn To Merge Experimental and Main Build Worker
- [x] Candidate NodeMCU v2, ESP32 and NodeMCU v3 as Testers (Matrix)
- [x] Make Special Commands Recognizable via Github API hooked from Github Action Runners
- [x] ~~Force Passing if no changes are handled even on Ignore Commit Mode.~~ (Its impossible under complex circumstances)
- [x] Fix badges, remove experimental when experimental and build workers were done.

All of these will be submitted in PR.

**Comment by CodexLink** [comment]
I gotta do some modifications first since we're going into the returnee state of this project. Here's the list:

- [ ] Recreate File Named Experimental Build Worker (filename may vary).
- [ ] Learn To Merge Experimental and Main Build Worker
- [ ] Candidate NodeMCU v2, ESP32 and NodeMCU v3 as Testers (Matrix)

All of these will be submitted in PR.

**Comment by CodexLink** [comment]
As of now, we have finished like 66.667% of the checklist. #29 is one of the indications of the development. I hope that I could close this one out by eliminating the things that are still needed to work on.

**Comment by CodexLink** [comment]
PR #30 fixes some flaws on the workflow. Though it doesn't resolve the 2nd to the last checklist which is quite impossible for me to get further. Continue-On-Error doesn't work, Force Return 0 doesn't stop the workflow. We then conclude, to accept failed remarks due to the fact that it automatically fails the workflow. Though it should fail because the workflow has nothing to do in return but nothing.

Closing as we have solved this issue for now. Congrats self.
